site/ui/components/main/Description.vue

```diff
@@ -35,13 +39,21 @@ export default {
   components: { VueMarkdown },
   data () {
     return {
+      truncated: true,
+      hidden: true,
```

Comment:
I don't think we need two booleans here. We seem to be always checking if they have the same value anyways.